pkg/driver/controller.go

```diff
@@ -92,7 +92,7 @@ func (cs *controller) CreateVolume(
 	logrus.Infof("CreateVolume: volume: {%v} is created", req.GetName())
 	return &csi.CreateVolumeResponse{
 		Volume: &csi.Volume{
-			VolumeId:      req.GetName(),
+			VolumeId:      utils.StripName(req.GetName()),
```

Comment:
why this change ?

Reply:
We are creating jivaVolume CR using the stripped name hence we need to send the volume ID as stripped for future references.